### ISSUE TITLE
bushslicer tests failing for OSA

### ISSUE THREAD

**Issue by kargakis** [openshift/openshift-azure#963]
```
[31mFailing Scenarios:[0m
[31mcucumber -p junit -p dir_embedder /home/jenkins/bushslicer/features/images/jenkins.feature:157[0m[90m # Scenario Outline: Trigger build of application from jenkins job with persistent volume, Examples (#2)[0m
[31mcucumber -p junit -p dir_embedder /home/jenkins/bushslicer/features/online/metrics.feature:5[0m[90m # Scenario: Ordinary user could view CPU,memory, network metrics statistics on pod page of openshift web console[0m
[31mcucumber -p junit -p dir_embedder /home/jenkins/bushslicer/features/quickstarts/quickstarts.feature:51[0m[90m # Scenario Outline: quickstart with persistent volume test, OS Type (#6)[0m
[31mcucumber -p junit -p dir_embedder /home/jenkins/bushslicer/features/web/overview.feature:181[0m[90m # Scenario: Check app resources on overview page[0m
[31mcucumber -p junit -p dir_embedder /home/jenkins/bushslicer/features/web/service-catalog/project.feature:5[0m[90m # Scenario: Manage project in popup panel on home page[0m
[31mcucumber -p junit -p dir_embedder /home/jenkins/bushslicer/features/web/settings.feature:145[0m[90m # Scenario: Check Openshift Master and Kubernetes Master version on About page[0m
```
https://openshift-gce-devel.appspot.com/build/origin-ci-test/logs/periodic-ci-openshift-openshift-master-azure-e2e-bushslicer/4/
https://openshift-gce-devel.appspot.com/build/origin-ci-test/logs/periodic-ci-openshift-openshift-master-azure-e2e-bushslicer/3/

@cjryan @kwoodson 
/kind bug
/area tests

**Comment by kwoodson** [comment]
@kargakis, thanks for bringing these up.  These were the failures we had previously seen.

Looks like jenkins failed to start so we definitely need to dig into the persistent volume test.

Trigger build of application from jenkins job with persistent volume, Examples
```
[31m      Jenkins failed to start within 300 seconds (RuntimeError)[0m
[31m      /home/jenkins/bushslicer/features/step_definitions/images/jenkins.rb:87:in `/^I have an?( ephemeral| persistent)? jenkins v([0-9]+|<%=.+?%>) application(?: from "(.+?)")?$/'[0m
[31m      /home/jenkins/bushslicer/features/images/jenkins.feature:157:in `And I have a jenkins v2 application'[0m
[31m      /home/jenkins/bushslicer/features/images/jenkins.feature:82:in `And I have a jenkins v<ver> application'[0m
```

These three tests threw `Not implemented` exception.

Ordinary user could view CPU,memory, network metrics statistics on pod page of openshift web console
Scenario: Manage project in popup panel on home page
Check Openshift Master and Kubernetes Master version on About page
```
      [36m[33m[16:48:48] INFO> Navigating to: https://e2e-ruycesft.eastus.cloudapp.azure.com/console/oauth#access_token=I0Fa1-7-DPm2b80NxIwMIpfiHjK41lPUs_rdvtFW5VQ&expires_in=86400&state=/&token_type=Bearer[0m[0m
[31m      Not implemented yet (RuntimeError)[0m
[31m      /home/jenkins/bushslicer/lib/webauto/web4cucumber.rb:125:in `browser'[0m
[31m      /home/jenkins/bushslicer/lib/webauto/web4cucumber.rb:435:in `goto_url'[0m
[31m      /home/jenkins/bushslicer/lib/webauto/web4cucumber.rb:195:in `block in run_action'[0m
[31m      /home/jenkins/bushslicer/lib/webauto/web4cucumber.rb:191:in `each'[0m
[31m      /home/jenkins/bushslicer/lib/webauto/web4cucumber.rb:191:in `run_action'[0m
[31m      /home/jenkins/bushslicer/lib/webauto/webconsole_executor.rb:115:in `login'[0m
[31m      /home/jenkins/bushslicer/lib/webauto/webconsole_executor.rb:144:in `run_action'[0m
[31m      /home/jenkins/bushslicer/lib/openshift/user.rb:105:in `webconsole_exec'[0m
[31m      /home/jenkins/bushslicer/features/step_definitions/web.rb:16:in `/^I run the :(.+?) web( console)? action$/'[0m
[31m      /home/jenkins/bushslicer/features/web/settings.feature:151:in `When I run the :goto_about_page web console action'[0m

```

As for the others I'm not exactly sure what happened.

@cjryan, Mind digging into these?

**Comment by kwoodson** [comment]
@cjryan and I looked at the persistent volume test failure.  We have determined that the jenkins pod takes > 5 minutes to start which times out the test and causes a failure.  The jenkins pod starts and the persistent volume mounts fine.  The jenkins web server takes a long time to allow logins.  We aren't sure exactly why this is occurring.  

Current plan is to increase timeout for jenkins to allow for it to settle before we start the test.
